src/js/components/epics/createForm.tsx

```diff
@@ -132,7 +132,7 @@ const CreateEpicModal = ({
     });
     /* istanbul ignore else */
     if (isMounted.current) {
-      setProjectBranches(baseBranches);
+      setProjectBranches(baseBranches || []);
```

Comment:
Hey, the story revealed an edge case in the actual code! 🎉 We weren't handling a potential `404` response for this request.

Reply:
🎊